modules/cryptable.rb

```diff
@@ -0,0 +1,36 @@
+module Cryptable
+
+  def index_to_alphabet
+    index_to_letter = @character_set.group_by do |letter|
+      @character_set.index(letter)
+    end
+
+    index_to_letter.transform_values! do |value|
+      value.join
+    end
+  end
+
+  def alphabet_to_index
+    index_to_alphabet.invert
+  end
+
+  def crypt_message(message, shifts)
+    crypted_message = ""
```

Comment:
Change not only method name but also local variable to make sense for both instances.